### PR TITLE
Add support for Python 3.5, 3.6, 3.7, 3.8 and 3.9.

### PR DESCRIPTION
**Pull request by icemac**
Drop support for Python 2.6 and 3.3.

Switch to GHA.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,40 +36,42 @@ def read(*rnames):
...
-    url = 'https://pypi.python.org/pypi/z3c.csvvocabulary',
-    packages = find_packages('src'),
-    include_package_data = True,
-    package_dir = {'':'src'},
-    namespace_packages = ['z3c'],
-    extras_require = dict(
-        test = ['zope.testing'],
-        ),
-    install_requires = [
+    url='https://pypi.python.org/pypi/z3c.csvvocabulary',
```
Comment (mgedmin) [inline setup.py]:
```suggestion
    url='https://https://github.com/zopefoundation/z3c.csvvocabulary',
```

File: setup.py [hunk 2]
```diff
@@ -34,40 +36,42 @@ def read(*rnames):
...
         'zope.i18nmessageid',
         'zope.schema',
-        ],
-    tests_require = ['zope.testing'],
-    test_suite = 'z3c.csvvocabulary.tests.test_suite',
-    zip_safe = False,
-    )
+    ],
+    tests_require=['zope.testing'],
+    test_suite='z3c.csvvocabulary.tests.test_suite',
```
Comment (mgedmin) [inline setup.py]: We could drop `test_suite=...`, python setup.py test is deprecated.
```suggestion
```

**Review by mgedmin — APPROVED** [review]
lgtm

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**